app/basicComponents/DropDown.tsx

```diff
@@ -260,6 +269,48 @@ const ItemsWrapper = styled.div<{
     `
       border-top: none;
   `};
+  top: 100%;
+`;
+
+const StyledDropDownItem = styled.div<{
+  uppercase?: boolean;
+  isLightSkin: boolean;
+}>`
+  width: 100%;
+  font-size: 13px;
+  ${({ uppercase }) =>
+    uppercase &&
+    css`
+      text-transform: uppercase;
+    `};
+  color: ${({
+    isLightSkin,
+    theme: {
+      form: {
+        dropdown: { dark, light },
+      },
+    },
+  }) => (isLightSkin ? dark.states.hover.color : light.states.hover.color)};
+  cursor: inherit;
+`;
+const DropDownLabel = styled.p<{
+  isBold?: boolean;
+  uppercase?: boolean;
+}>`
+  font-weight: ${({ isBold }) => (isBold ? 800 : 400)};
+  font-size: 14px;
+  text-transform:  ${({ uppercase }) => (uppercase ? 'uppercase' : 'inherit')};
+}
+`;
+
+const DropDownLabelDescription = styled.p<{
+  isBold?: boolean;
+}>`
+  font-weight: ${({ isBold }) => (isBold ? 800 : 400)};
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-top: 10px;
```

Comment:
```suggestion
  margin-top: 6px;
```
<img width="568" alt="image" src="https://user-images.githubusercontent.com/1897530/176413308-3ee0f6b4-58fc-404d-b385-5422be4f2d0d.png">